tests/refs/reflog/messages.c

```diff
@@ -37,34 +37,34 @@ void test_refs_reflog_messages__setting_head_updates_reflog(void)
 	cl_git_pass(git_repository_set_head(g_repo, "refs/tags/test"));            /* 1 */
 	cl_git_pass(git_repository_set_head(g_repo, "refs/remotes/test/master"));  /* 0 */
 
-	reflog_check_entry(g_repo, GIT_HEAD_FILE, 4,
```

Comment:
I'd prefer if these whitespace changes were put inside another commit, which _only_ changes indentantion. Furthermore, instead of adding more indentation, I'd break with our usual style here and decrease indentation to only one tab on the newline. Otherwise, lines get very long and the statements become quite unreadable.